### PR TITLE
[HCGOVDS-6] Moving error message below date field

### PR DESCRIPTION
**Pull request by laurellawrence**
Updating the HTML examples for date field to move the inline error messages to the bottom

### CODE REVIEW

File: docs/src/pages/components/DateField/_DateField.docs.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@uswds https://designsystem.digital.gov/components/form-controls/#date-input
+
+Three text fields are the easiest way for users to enter most dates.
+
+Markup: datefield.example.html
+
+Style guide: components.date-field
+*/
+
+/*
```
Comment (line47) [inline docs/src/pages/components/DateField/_DateField.docs.scss:13]: I'm my testing it looks like we only need to keep the top portion of this page and the rest will be inherited from the core CMS Design System. I think you can remove line 13 to the end and push the PR back up

**Review by line47 — CHANGES_REQUESTED** [review]
I think we can remove the content that is not changing

File: docs/src/pages/components/DateField/_DateField.docs.scss (hunk 1 above)
Comment (laurellawrence) [reply docs/src/pages/components/DateField/_DateField.docs.scss]: Line 13 and beyond has been removed and ready for review

**Review by line47 — APPROVED** [review]
Looks good to me!

**Review by nichia — COMMENTED** [review]
This looks great. 
I wonder if we can also include the link to `CMS Design System`, and add a newline at the end of the example file.

File: docs/src/pages/components/DateField/datefield.example.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+              value="2050"
+            />
+          </div>
+        </div>
+        <span class="ds-c-field__error-message ds-c-field__error-message--inverse" role="alert">
+            Please enter a year in the past
+          </span>
+      </fieldset>
+    </div>
+  </div>
```
Comment (nichia) [inline docs/src/pages/components/DateField/datefield.example.html:114]: Add a newline to end of this file
```suggestion
  </div>
  
```

**Comment by line47** [comment]
> I wonder if we can also include the link to CMS Design System, and add a newline at the end of the example file.

@nichia I think we might need to add a CMSDS flag to accomplish that? 
https://github.com/CMSgov/design-system/blob/master/guides/WRITING-DOCUMENTATION.md#flags

**Review by nichia — APPROVED** [review]
Looks like there may not be a flag to easily add the cmsds link, so there's no need to add that then. This PR is good to go!

**Pull request merged**